docs/Spring Boot/5_Step_To_Make_Gradle_Configuration_Extreme_Clean_In_A_Multi-Module_Project.md

```diff
@@ -0,0 +1,216 @@
+# 5 Step to Make Gradle Configuration Extreme Clean in a Multi-Module Project
```

Comment:
⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Readability.FleschKincaid] Try to keep the Flesch–Kincaid grade level (9.59) below 8.